og.module

```diff
@@ -175,6 +191,11 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
 function og_entity_create_access(AccountInterface $account, array $context, $bundle) {
   $entity_type_id = $context['entity_type_id'];
 
+  if (!empty($context['skip_og_permission'])) {
+    // We've already been here or we want to skip the access check.
+    return;
```

Comment:
I would try to see if it is possible to find a solution that avoids adding this section. Since this hook is dealing with checking access it is important for security reasons and introducing a way to skip the checks might have unintended consequences.

It is also not allowed to return `NULL` here according to the entity API, we should always return an `AccessResult`.